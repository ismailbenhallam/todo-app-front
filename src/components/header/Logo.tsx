import styled from "styled-components";
import logo from "../../logo.webp";

const Img = styled.img`
  /* width: 40px; */
  height: 100%;
`;

export default function Logo() {
  return <Img src={logo} alt="Logo" />;
}
