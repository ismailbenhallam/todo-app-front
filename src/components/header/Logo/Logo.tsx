import logo from "logo.webp";
import { FC } from "react";
import styled from "styled-components";

const LogoContainer = styled.img`
  height: 100%;
  max-height: 70px;
`;

const Logo: FC = () => {
  return <LogoContainer src={logo} alt="Todo App Logo" />;
};

export default Logo;
