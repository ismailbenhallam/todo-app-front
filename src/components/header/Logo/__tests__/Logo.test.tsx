import renderer from "react-test-renderer";
import Logo from "..";

test("Logo should match snapshot", () => {
  const tree = renderer.create(<Logo />).toJSON();
  expect(tree).toMatchSnapshot();
});
