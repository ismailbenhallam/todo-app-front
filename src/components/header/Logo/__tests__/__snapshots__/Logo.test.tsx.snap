// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo should match snapshot 1`] = `
<img
  alt="Todo App Logo"
  className="sc-bdfBQB gldXoE"
  src="logo.webp"
/>
`;